CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.3.0 - 2017-5-3
+This is a feature release
+  * (SERVER-1772) Add support for building and installing additional uberjars.
+    An `:additional-uberjars` EZBake setting has been added which allows projects
+    to specify a list of versioned dependencies that will be built and installed
+    next to the projects own uberjar.
+  * (SERVER-1772) Support installing a `cli-defaults.sh` file which can be
+    configured by EZBake projects and supply defaults for bash variables in a
+    project's cli scripts
+
```

Comment:
Looks like some work for [EZ-108](https://tickets.puppetlabs.com/browse/EZ-108) landed since the 1.2.1 release, too.  Seems like we should mention something about that, too.